### ISSUE TITLE
Does not work on Quest 2

### ISSUE THREAD

**Issue by Bluscream** [tverona1/QuestAppLauncher#63]
On my quest 2 it always crashes :(

**Comment by hjochman** [comment]
Hi,

check if you have two packages installed with the same package name but different big and small letters. That was my problem

23. Februar 2021 11:31, "Bluscream" <notifications@github.com (mailto:notifications@github.com?to=%22Bluscream%22%20<notifications@github.com>)> schrieb:
	On my quest 2 it always crashes :(

**Comment by hollmrl** [comment]
I am having the same issue also.  It works fine on my Quest 1, but not my Quest 2.  I have checked all installed apk's and there are not any duplicates with changes to upper/lower case letters so there must be some other issue.  I am not sure how to obtain log files to see what the error is.  If anyone else has had success with the Quest 2 I would be interested to know.

**Comment by WilsonBradley** [comment]
Same...

Instructions are pretty vague to say the least..

**Comment by WilsonBradley** [comment]
1st Question.. Is the latest available build posted on SideQuest suppose to NOT have any icons?
I have done this so many times now.. Wiped any trace and reinstall on Quest 2.

If so, then
The instructions to install Assets are vague and hard to follow.
We need a video tut or something.

**Comment by WilsonBradley** [comment]
Countless hours wasted on trying to get this to work without success.
Could someone point to some clear instructions since it doesn't work right "out of the Box"

Vague Details here..
https://github.com/reloxx13/QuestAppLauncher_Assets/discussions/3

**Comment by WilsonBradley** [comment]
UPDATE:
So turns out, something is corrupt / wrong with my Quest 2 install of QuestAppLauncher. I verified that it is not standard behavior to have no icons at start.
I have removed data/cache from app, completely removed aaa.QuestAppLauncher folder
android/data/aaa.QuestAppLauncher.App/
Uninstalled QuestAppLauncher

Any other suggestions?
![109707627-01128580-7b60-11eb-89be-dc3bc829e333](https://user-images.githubusercontent.com/8173893/110951570-d6b48b00-830a-11eb-804f-e251abcedb5e.jpg)
![109707620-fd7efe80-7b5f-11eb-8fac-10fbf64a16f2](https://user-images.githubusercontent.com/8173893/110951575-d916e500-830a-11eb-9ddc-1912ae419d15.jpg)
![109707608-fa840e00-7b5f-11eb-9b79-462ccc3e23d3](https://user-images.githubusercontent.com/8173893/110951581-dae0a880-830a-11eb-80c8-e47c4a34a119.jpg)

**Comment by WilsonBradley** [comment]
> I am having the same issue also. It works fine on my Quest 1, but not my Quest 2. I have checked all installed apk's and there are not any duplicates with changes to upper/lower case letters so there must be some other issue. I am not sure how to obtain log files to see what the error is. If anyone else has had success with the Quest 2 I would be interested to know.

Ever get this working..? See my issue above.

**Comment by hollmrl** [comment]
Unfortunately no.  I have not had the time to explore the issue further.

**Comment by reloxx13** [comment]
Get the android sdk platform tool https://developer.android.com/studio/releases/platform-tools   
enable dev mode for ur headset    
connect your headset to pc   
run `adb logcat` or for nice output `adb logcat -b all -v color` in a command line from the platform tools folder.   
start the QAL app and watch the logs for errors.

**Comment by WilsonBradley** [comment]
Tha KS for the reply.. I couldn't wait any longer and did a factory reset..
All working now.. Thanks
